cnd/src/asset/mod.rs

```diff
@@ -36,7 +25,7 @@ impl Asset for Ether {}
 
 impl Asset for Erc20 {}
 
-#[derive(Clone, Copy, Derivative, PartialEq)]
```

Comment:
Sad that all of these `Copy` implementations go away :(

Reply:
I know!